ckanext/dashboard/blueprints/dashboard.py

```diff
@@ -0,0 +1,115 @@
+import logging
+from ckan import model
+import ckan.plugins as p
+from flask import Blueprint, request, redirect, url_for
+from ckan.logic import NotFound
+from ckan.plugins.toolkit import render
+from ckan.lib.helpers import helper_functions as h
+
+# Import or define the decorator to restrict access to sysadmins.
+# You can define it in your extension or import it if you already have it.
+from ckanext.dashboard.decorators import require_sysadmin_user
+
+log = logging.getLogger(__name__)
+
+dashboard_bp = Blueprint('dashboard_bp', __name__, url_prefix='/dashboard-external')
```

Comment:
```suggestion
dashboard_bp = Blueprint('embeded_dashboard', __name__, url_prefix='/embeded-dashboard')
```

Since `dashboard` is already taken in CKAN core, as @avdata99 mentioned, I will suggest to use `embeded_dashboard` and `/embeded-dashboard` since it is more explicit than `external`.

Embeded Dashboards is a common terminology in the BI and Analytics world.